.github/workflows/build_images.yml

```diff
@@ -2,44 +2,43 @@ name: Build Images
 
 concurrency: ${{ github.ref }}
 
-on: 
+on:
   push:
     branches: [main]
     paths:
-      - 'images/**'
-      - 'scripts/**'
+      - "images/**"
+      - "scripts/**"
   workflow_dispatch:
 
 jobs:
-
   build:
     runs-on: ubuntu-latest
 
     if: "!contains(join(github.event.commits.*.message), '+nobuild')"
 
     env:
-      RESOURCE_GROUP_NAME: aqovia-devbox
-      IDENTITY_NAME: aqovia-devcenter-identity
-      GALLARY_NAME: aqovia_compute_gallery
-      VNET_NAME: aqovia-devbox-builder-vnet
+      RESOURCE_GROUP_NAME: dlr-devbox
+      IDENTITY_NAME: dlr-devcenter-identity
+      GALLARY_NAME: dlr_compute_gallery
+      VNET_NAME: dlr-devbox-builder-vnet
       SUBNET_NAME: default
-      STORAGE_ACCOUNT_NAME: aqdevboxbuilderstrg
+      STORAGE_ACCOUNT_NAME: dlrdevboxbuilderstrg
       LOCATION: westeurope
 
     steps:
       - uses: actions/checkout@v2
 
       - name: Login to Azure
-        run: az login --service-principal -u ${{ secrets.AZURE_CLIENT_ID }} -p ${{ secrets.AZURE_CLIENT_SECRET }} --tenant ${{ secrets.AZURE_TENANT_ID }}
-          
+        run: az login --service-principal -u="${{ secrets.AZURE_CLIENT_ID }}" -p="${{ secrets.AZURE_CLIENT_SECRET }}" --tenant="${{ secrets.AZURE_TENANT_ID }}"
```

Comment:
I think we need to change `AZURE_CLIENT_ID` and `AZURE_TENANT_ID` to be simple environment variables and no secrets. Can you change that please? I am going to do the change in repository settings.

Reply:
I just updated with new values. It will also use DLR-Product-Interconnection-DevTest subscription.

Reply:
done